numpy/lib/tests/test_arraysetops.py

```diff
@@ -871,10 +870,7 @@ def test_unique_axis(self):
         types = []
         types.extend(np.typecodes['AllInteger'])
         types.extend(np.typecodes['AllFloat'])
-        types.append('datetime64[D]')
-        types.append('timedelta64[D]')
-        types.append([('a', int), ('b', int)])
-        types.append([('a', int), ('b', float)])
+        types.extend(('datetime64[D]', 'timedelta64[D]', [('a', int), ('b', int)], [('a', int), ('b', float)]))
```

Comment:
The original is easier to read and understand. The new line is also a touch long.

Reply:
I can fold the line to match the limit of 88 characters, or even change to something like this:
```python
        types.extend((
            'datetime64[D]',
            'timedelta64[D]',
            [('a', int), ('b', int)],
            [('a', int), ('b', float)],
        ))
```
But then if Numpy is ever formatted by a formatter like _ruff_, the above will be folded back into two lines.